bundle.Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM scratch
 
 # RH OLM annotations
-LABEL com.redhat.openshift.versions="v4.5-v4.8"
+LABEL com.redhat.openshift.versions="v4.5,v4.6"
```

Comment:
Why change to v4.5 and v4.6? Does it mean it would work for v4.6+?

Reply:
with this change, the operator would change from supporting ocp versions `4.5-4.8`, to instead supporting ocp `4.5+`